tools/slack_tools/collect_prs.py

```diff
@@ -0,0 +1,192 @@
+# Script for collecting PRs in need of review, and informing maintainers via
+# slack
+
+from __future__ import print_function
+
+import datetime
+import os
+import sys
+
+import github
+
+try:
+    input = raw_input  # Python 2
+except NameError:
+    pass  # Python 3
+
+maintainers = {
```

Comment:
as i think this is a constant best to capitalize (for bonus points use a tuple as its immutable and consume to dict later)